tests/snmpsim/fortios.snmprec

```diff
@@ -1 +1,41 @@
+1.3.6.1.2.1.1.1.0|4|
```

Comment:
This is perfect with the exception of sysDescr - is it genuinely blank?

Reply:
It's blank unless you manually set it.

Reply:
Ok, thanks :)